backend/src/main/java/wooteco/retrospective/presentation/attendance/AttendanceController.java

```diff
@@ -24,8 +33,27 @@ public AttendanceController(AttendanceService attendanceService) {
     @PostMapping("/time")
     public ResponseEntity<AttendanceResponse> postTime(@Valid @RequestBody AttendanceRequest attendanceRequest) {
         AttendanceResponse attendanceResponse = AttendanceResponse.of(
-                attendanceService.postAttendance(attendanceRequest));
+            attendanceService.postAttendance(attendanceRequest));
 
         return ResponseEntity.ok().body(attendanceResponse);
     }
+
+    @DeleteMapping("/time")
+    public ResponseEntity<Void> deleteTime(@Valid @RequestBody AttendanceRequest attendanceRequest) {
+        attendanceService.deleteAttendance(attendanceRequest);
+
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping("/time/{time}")
+    public ResponseEntity<AttendanceByTimeResponse> getTime(@PathVariable("time") long timeId) {
+        Time time = attendanceService.findTimeById(timeId);
+        List<Member> members = attendanceService.findAttendanceByTime(time);
+
+        AttendanceByTimeResponse attendanceByTimeResponse = AttendanceByTimeResponse.of(
+            time, members
+        );
```

Comment:
```suggestion
        AttendanceByTimeResponse attendanceByTimeResponse =
                AttendanceByTimeResponse.of(time, members);
```

저는 `=`이 윗라인에 위치해야 한다고 생각해요 !